moviedb_project/moviemanager/views.py

```diff
@@ -2,23 +2,35 @@
 from django.http import HttpResponse, HttpResponseRedirect,JsonResponse
 from .forms import NameForm
 from .models import NameFormModel
-import json, os,requests
+import json, os, requests
+from datetime import datetime
 # Create your views here.
 database = []
 def moviemanagerView(request):
     if request.method == 'POST':
         form = NameForm(request.POST)
         #year_model = Year(request.POST)
         if form.is_valid():
-            database.append(form.cleaned_data)
-            print(database)
-            print(type(database))
+            #Not initializing db here, instead storing cleaned data for manipulation
+            cleanForm = form.cleaned_data
+            print(cleanForm)
+            print(type(cleanForm))
             #obj = NameFormModel.objects.create(**form.cleaned_data)
-            form = NameForm()
-            movies_json = JsonResponse(database, safe = False)
+            movies_json = JsonResponse(cleanForm, safe = False)
+            #Always have to futz with dates for one reason or another. This prevents the db from having datetime() instead of your actual date.
+            cleanForm["date_watched_field"] = cleanForm["date_watched_field"].strftime("%m/%d/%Y")
+            #Check to see if your db already exists, if it doesn't you need to do some more work. If it does you can just continue
+            if os.path.exists('moviedatabase.json'):
+                with open('moviedatabase.json') as outfile:
+                    data = json.load(outfile)
+            if not os.path.exists('moviedatabase.json'):
+                data = []
+            data.append(cleanForm)
             with open('moviedatabase.json','w') as outfile:
-                json.dump(database,outfile)
-            return render (request, 'form.html',{'form':form})
+                json.dump(data,outfile)
+            # This is just to show a response to the user. You can render another form here instead of the default form. This is your chance to return a templated list instead but this should get you on the right road.
+            return HttpResponse(str(data))
```

Comment:
Just changing to a different return so you can see the DB results. You'll likely want to make a template for this result in some kind of pretty table